exasol/ai/mcp/server/mcp_server.py

```diff
@@ -109,11 +116,13 @@ def _register_tools(self):
                 self.describe_table,
                 description=(
                     "Describes the specified table in the specified schema of the "
```

Comment:
```suggestion
                    "This tool describes the specified table in the specified schema of the "
```

Reply:
Then, I guess all tools' descriptions should start from "This tool describes ..."